ui/src/app/views/notebook-list/component.html

```diff
@@ -42,4 +72,7 @@ <h3 class="modal-title">Error:</h3>
     <button type="button" class="btn btn-primary" (click)="localizeNotebooksError = false">Ok</button>
   </div>
 </clr-modal>
-<app-bug-report></app-bug-report>
\ No newline at end of file
+<app-bug-report></app-bug-report>
+
+<app-rename-modal [resourceType]="'notebook'" [resource]="notebookInFocus" (receiveRename)="receiveRename($event)"></app-rename-modal>
+<app-confirm-delete-modal [resourceType]="'notebook'" [resource]="notebookInFocus" (receiveDelete)="receiveDelete($event)"></app-confirm-delete-modal>
```

Comment:
Nit: newline

Reply:
yup, wasn't quite ready for reviews. Sorry!